core/src/test/resources/specmaticConfigFiles/specmatic.yml

```diff
@@ -55,4 +55,11 @@ security:
         value: API-QUERY-PARAM-USER
       BasicAuth:
         type: basicAuth
-        token: Abc123
\ No newline at end of file
+        token: Abc123
+
+resiliencyTests:
+  enable: all
+  maxTestRequestCombinations: 50
+enableResponseValueValidation: true
+enableExtensibleSchema: true
+enableSchemaExampleDefault: false
```

Comment:
This is a mechanical name. Should we come up with alternative, e.g. `useSchemaDefaultInTests`

Reply:
This makes sense. Will update the name.